src/main/java/ua/softserveinc/tc/service/impl/DayOffServiceImpl.java

```diff
@@ -42,139 +39,106 @@
     private UserService userService;
 
     @Autowired
-    private EventRepository eventRepository;
+    private EventService eventService;
 
     @Autowired
-    private DayOffRepository dayOffRepository;
-
-    /**
-     * Creates {@link DayOff} in database
-     * and sends information email to users, if
-     * there is less than seven days till day off,
-     * creates it in calendar for appropriate rooms
-     *
-     * @param dayOff a requested day off
-     * @return current day
-     */
+    private DayOffDao dayOffDao;
+
+    @Log
+    private static Logger log;
+
     @Override
-    public DayOff create(DayOff dayOff) {
+    public void create(DayOff dayOff) {
```

Comment:
This method create() is implemented in BaseDao class and yes, create method in base class is safe. Why I return void? Because returning of DayOff object is never used, so it is safe and important to return void.